### ISSUE TITLE
[ReSpec] The "Publication" term seems to by synonymum for "Dataset"

### ISSUE THREAD

**Issue by vlcinsky** [mobilityDCAT-AP/mobilityDCAT-AP#52]
See the definitions in the document https://mobilitydcat-ap.github.io/mobilityDCAT-AP/releases/1.0.1/index.html#terminology

> A **Dataset** is a collection of data published or curated by a single source and available for access or download in one or more formats. In the mobility context, this might be, e.g., a data collection about static parking information for truck drivers published by a road authority.

and then

> A **Publication** is an abstract construct that covers a dataset published by a data publisher and which has one or multiple distributions.

While Dataset is widely used, I did not find any specific use of Publication term.

I did not find any definition for Publication in DCAT-AP v2 and DCAT-AP v3.

Consider removing this term from mobilityDCAT-AP.

**Comment by andreyfmc** [comment]
I think this will help you.
https://bit.ly/4gwC82r
If you don't have the c compliator, install it.(gcc or clang)

**Comment by vlcinsky** [comment]
**WARNING** the [comment above](https://github.com/mobilityDCAT-AP/mobilityDCAT-AP/issues/52#issuecomment-2360731972) is possibly spam and might lead to malicious URL.

The reply apparently does not relate to the question and the user has almost empty history.

**Comment by marioscrock** [comment]
Thank you @vlcinsky for opening the issue and reporting the malicious comment above (I deleted it). We will discuss this topic during the next mobilityDCAT-AP meeting and get back to you.

**Comment by EdNDW** [comment]
In the Netherlands, we distinguish between publications and datasets. A publication is a description of one or more datasets on the same subject. A dataset is a specific version in a particular format. For example, in the Dutch NAP, there is a description of one real-time publication related to travel times. This publication contains various datasets in formats such as XML and JSON.

**Comment by vlcinsky** [comment]
Thanks @EdNDW for feedback.
Terminology is hard but important topic.

We have created study [DATEX@NAPs](https://datex2.naps.inqms.tamtamresearch.com) and struggled with terminology. Almost every institution or publisher has their own terminology, e.g.:

- feed, source, publication, distribution, dataset
- publisher, provider

For that reason we searched for some common ground and found that mobilityDCAT-AP is the one - it is maintained, updated and relates to mobility data.

Our terminology based on mobilityDCAT-AP is describe here: [DATEX@NAPs/Key Concepts](https://datex2.naps.inqms.tamtamresearch.com/key-concepts.html)

I have to say, it is not easy to stick with any agreed terminology - the habit of "our own best terms" is hard to overcome.

Regarding terms "dataset" and "publication" you have described:
- mobilityDCAT-AP uses "dataset" to what you call a publication
- mobilityDCAT-AP uses "distribution" to what you call a dataset

It would propose to organize a workshop to discuss and clarify the moblityDCAT-AP terms. Ideally reviewing some text which we really use for talking about the data we provide.

**Comment by marioscrock** [comment]
I agree on removing the term `Publication` as it is confusing (not sure the motivation why it was added) and in general we may want to simplify the Terminology section as in [DCAT-AP](https://semiceu.github.io/DCAT-AP/releases/3.0.0/#specterminology). 
I agree with the comment from @vlcinsky, but considering the comment from @EdNDW I would like to specify that all the `Distribution` of a `Dataset` should contain exactly the same information (possibly in different formats or accessible in different ways). If we consider different "versions" of the data, these are different `Datasets`. 

In DCAT-AP v3.0.0, the concept of `dcat:DatasetSeries` is introduced to represent "logically" related datasets (cf. [here](https://semiceu.github.io/DCAT-AP/releases/3.0.0/#usage-guide-on-dataset-series) for a more detailed definition and explanation). The inclusion of this new class is currently under discussion for mobilityDCAT-AP v3.0.0.

**Comment by EdNDW** [comment]
@marioscrock @vlcinsky I agree that I have used the wrong definitions earlier. From my perspective this should be the correct definitions according to DCAT: 

**Dataset** represents the conceptual level - it describes a logical collection of data about a particular subject. It is abstract and independent of how the data is actually made available. For example: "Population Statistics Netherlands 2023" as a dataset.
**Distribution** represents the concrete level - this describes how a dataset is actually made accessible. A single dataset may have multiple distributions in different formats, via different access methods, or with different licences. For example: the same population statistics might have distributions as a CSV download, JSON via API, or Excel file.
**Publication** (or dcat:DataService in DCAT-AP) refers to the service or mechanism through which distributions are offered. This could be an API, download service, or data portal that provides access to one or more distributions.
The hierarchy is therefore:

Dataset (what) → Distribution (how) → Publication/DataService (where/via which service)

In line with this, **Dataset Series** (or dcat:DatasetSeries) is an important concept that sits between the catalogue and individual datasets in the DCAT hierarchy.
A Dataset Series groups related datasets that together form a logical sequence, usually organised around:

- **Time**: for example, monthly unemployment figures, annual population statistics
- **Geographical areas**: the same type of data for different regions
- **Versions**: successive versions of the same type of dataset

The hierarchy then becomes:

Catalogue (the complete data catalogue)
     Dataset Series (for example "Timetables 2020-2025")
          Dataset (for example "Timetable 2023")
               Distribution (CSV, JSON, Excel versions)
                    Publication/DataService (download portal, API)

**Comment by peterlubrich** [comment]
I am sorting things out:
1. The "Publication" terms comes from the good, old days of the Coordinated Metadata Catalogue. Semantically, it was the same as a "Dataset" in the DCAT-AP world
2. Ed is proposing to use "Publication" as a synonym with "Data Service".
3. DCAT-AP is using "publication" as a release process for a dataset, see  [here](https://semiceu.github.io/DCAT-AP/releases/3.0.1/#Dataset.releasedate)

Okay, to avoid any further confusion: I propose to give up the term "Publication" for now.
I cleaned up the [Terminology section](https://mobilitydcat-ap.github.io/mobilityDCAT-AP/drafts/latest/index.html#terminology).

**Comment by peterlubrich** [comment]
The wording issue has been resolved while revising the [Terminology section](https://mobilitydcat-ap.github.io/mobilityDCAT-AP/drafts/latest/index.html#terminology), see also issue #51
